recipe/conda_build_config.yaml

```diff
@@ -1,9 +1,9 @@
 c_stdlib_version:          # [osx and x86_64]
-  - "10.15"                # [osx and x86_64]
+- '10.15'                  # [osx and x86_64]
 suffix:
-  - ""
-  - "-novec"
```

Comment:
What is `suffix` supposed to achieve? I see [nothing](https://github.com/search?q=repo%3Aconda-forge%2Fonnxruntime-feedstock+suffix+path%3Arecipe&type=code) in the recipe that would justify building everything twice, which is already an enormous waste of resources in itself, but simply unacceptable when moving to a very limited resource like the opengpu server.

If the only goal is to do
https://github.com/conda-forge/onnxruntime-feedstock/blob/3185a04ffc9e7a8664eee41b9dee8986eca22878/recipe/meta.yaml#L107-L108

then this should be relegated to a meta-package that just wraps the main library (which should get built only once!), and then we only duplicate the build of the output that switches the constraint on or off.

Reply:
The suffix affects the package name, which then gets picked up in https://github.com/conda-forge/onnxruntime-feedstock/blob/3185a04ffc9e7a8664eee41b9dee8986eca22878/recipe/build.sh#L7. That means we're compiling with different eigen flags. `novec` gives guarantees that the same code paths is used in eigen regardless of the input size.

I'm not sure if that's already the case, but I don't see much need to have a `novec` version built for GPUs.

Reply:
OK, I did not notice that switch. If they're compiled in different ways then I guess that's understandable.

Reply:
Ah, I suppose they are still rendered but skipped at build time.